src/infer_request.cc

```diff
@@ -68,14 +68,15 @@ InferRequest::InferRequest(
     }
   }
 
-  inputs_ = inputs;
-  requested_output_names_ = requested_output_names;
+  inputs_ = inputs;                                  // TODO: do we need this?
```

Comment:
Why do we need `TODO`? It is clearly required.

Reply:
The two variables are already initialized to the ones on the constructor argument, i.e.
```c++
InferRequest::InferRequest(...) : ..., inputs_(inputs), requested_output_names_(requested_output_names), ...
{
  ...
  inputs_ = inputs;
  requested_output_names_ = requested_output_names;
}
```
I am not sure why we are assigning them again? The second copy seems unnecessary?

Reply:
Looks redundant to me when you put it like that :nerd_face: 

This PR LGTM, I'm fine with removing this redundancy if compilation and tests all look good :+1:

Reply:
Should be able to remove

Reply:
I see, yes, we can remove these. I thought Jacky was referring to the attributes themselves (i.e. remove `inputs_` and `requested_output_names_` and not the copy)..

Reply:
Removed: [[chore] Remove redundant copy](https://github.com/triton-inference-server/python_backend/pull/366/commits/b745ae54906a2b36c72a6f5a29b38411f99f06f9)